src/views/Challenge.vue

```diff
@@ -88,8 +82,24 @@ export default {
         { k: { v: 'title', r: true}, l: this.$t('challenge.ZA') },
         { k: { v: no_ass, r: false}, l: this.$t('challenge.NO_REVIEWS_DESC') },
         { k: { v: no_ass, r: true}, l: this.$t('challenge.NO_REVIEWS_ASC') },
-        { k: { v: 'random', r: false}, l: this.$t('challenge.RANDOM') },
+        { k: { v: 'random', r: false}, l: this.$t('challenge.RANDOM') }
       ]
+      const walletsCondition = this.proposals.filter((prop) => prop.votes_cast).length > 0;
+      if(walletsCondition){
+        options.push(
```

Comment:
Instead of using `options.push` you can use the spread operator `...` in the main object in combination with a condition.
Something like:
```
const default_obj = [
  { k: { v: 'amount', r: false}, l: this.$t('challenge.REQUESTED_BUDGET_DESC')},
  ...(this.withVoteCast ? [{ k: { v: 'votes_cast', r: false}, l: this.$t('challenge.WALLETS_DESC') }] : [])
]
```
This will keep the code more compact and the all the possible conditions in one place.